conference/src/test/java/com/kyriba/conference/management/service/HallServiceImplSpringTest.java

```diff
@@ -0,0 +1,81 @@
+package com.kyriba.conference.management.service;
+
+import com.kyriba.conference.management.dao.HallRepository;
+import com.kyriba.conference.management.domain.dto.HallRequest;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.validation.ValidationAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.validation.ConstraintViolationException;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = { HallServiceImpl.class, ValidationAutoConfiguration.class })
+public class HallServiceImplSpringTest
```

Comment:
I think, this should be renamed to ...SpringBootTest

Reply:
done